tests/test_lint.py

```diff
@@ -22,10 +22,27 @@ def pf(wd, path):
 WD = os.path.dirname(__file__)
 PATH_CRITICAL_EXAMPLE =  pf(WD, 'lint_examples/critical_example')
 PATH_FAILING_EXAMPLE = pf(WD, 'lint_examples/failing_example')
+PATH_WORKING_EXAMPLE = pf(WD, 'lint_examples/minimal_working_example')
 
 class TestLint(unittest.TestCase):
     """Class for lint tests"""
 
+    def test_call_lint_pipeline(self):
+        """Test the main execution function of PipelineLint
+        This should not result in any exception for the minimal
+        working example"""
+        lint_obj = nf_core.lint.PipelineLint(PATH_WORKING_EXAMPLE)
+        lint_obj.lint_pipeline()
+        assert len(lint_obj.failed) == 0, "Expected 0 missing file FAIL, but found %r" % len(lint_obj.failed)
+        assert len(lint_obj.warned) == 0, "Expected 0 missing file WARN, but found %r" % len(lint_obj.warned)
+        assert len(lint_obj.passed) == 14, "Expected 14 missing file PASS, but found %r" % len(lint_obj.passed)
```

Comment:
We should probably wrap these three `assert` statements into a function as we check them so often. But can do that in a future PR. eg. `check_lint_results(0,0,14)` or `check_lint_results(1, None, None)` etc.

Reply:
totally agree!